database_design.txt

```diff
@@ -0,0 +1,87 @@
+E-Commerce Database
+---------------------------
+This database contains customer table, shipping address table, 
+invoice table, payment table, product table, and order item table
+
+Keyword:
+PK --> Primary Key
+FK --> Foreign Key
+
+Customer Table
+Purpose: This table stores information about a customer
+
+Fields:
+    - customer id (PK)
+    - first name
+    - last name
+    - email address
+    - phone number
+    - shipping adresss (FK)
+    - credit limit
+    - invoice
+    - payment
+    - created date
+    - modified date
+
+Shipping Address Table
```

Comment:
yes it makes sense to separate addresses to its own table, since a customer may want to manage their addresses and reuse them when checking out. Maybe explicitly document this use case in the doc as well.